src/clarity-angular/button/index.ts

```diff
@@ -4,7 +4,7 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import {Type} from "@angular/core";
-import {Button} from "./button-group/button";
-import {ButtonGroup} from "./button-group/button-group";
+import {ClrButton as Button} from "./button-group/button";
+import {ClrButtonGroup as ButtonGroup} from "./button-group/button-group";
```

Comment:
That's not what you want. I get that you're trying to avoid breaking changes, but the whole point is to deprecate _and_ offer the new standard, not just deprecate. Otherwise people have nothing to move to. As a side note, you apparently got unlucky again and took (one of?) the only module(s) that didn't export the directives by themselves, when it should.

All in all, this file should be:
```typescript
import {Type} from "@angular/core";
import {ClrButton} from "./button";
import {ClrButtonGroup} from "./button-group";

export {ClrButton} from "./button";
export {ClrButtonGroup} from "./button-group";

export const CLR_BUTTON_GROUP_DIRECTIVES: Type<any>[] = [ClrButton, ClrButtonGroup];

/* tslint:disable */
/**
 * @deprecated
 */
export const Button = ClrButton;
/**
 * @deprecated
 */
export const ButtonGroup = ClrButtonGroup;
/**
 * @deprecated
 */
export const BUTTON_GROUP_DIRECTIVES = CLR_BUTTON_GROUP_DIRECTIVES;
```

And this is what I get when I try to use them:
<img width="300" alt="screen shot 2017-11-21 at 20 40 57" src="https://user-images.githubusercontent.com/4647197/33105664-55f8e5bc-cefc-11e7-8573-f7f3fbfcc4ce.png">

Reply:
@youdz: We don't want to introduce these breaking changes in 0.11. We have a very specific scope of breaking changes for 0.11. So if we are not deprecating, we will need to hold off to merge this in until 0.12.

I'm not sure how this jumped up in prioritiy. I thought we were clear on what was going into 0.11.

Reply:
That's the whole point of my comment, this is not a breaking change, just a deprecation.
It allows us to warn people that 0.12 will include the removal of these deprecated exports as a breaking change.

Whether this is 0.11.0, 0.11.1 or later, I don't think it matters. But the point is to have it early in 0.11.x, to give people a full minor's notice.

Reply:
I did this PR as a quick prototype so it could provide the pattern for the rest of the work. I was also thinking it needs to be as early as possible.